includes/configuration_managers/class-jetpack-configuration-manager.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Jetpack plugin configuration manager.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+require_once NEWSPACK_ABSPATH . '/includes/configuration_managers/class-configuration-manager.php';
+
+/**
+ * Common functionality for admin wizards. Override this class.
+ */
+class Jetpack_Configuration_Manager extends Configuration_Manager {
+
+	/**
+	 * The slug of the plugin.
+	 *
+	 * @var string
+	 */
+	public $slug = 'jetpack';
+
+	/**
+	 * Get this wizard's name.
+	 *
+	 * @return string The wizard name.
+	 */
+	public function configure() {
+		return true;
```

Comment:
Nitpicks: The docblock for `configure` in the abstract class says it shouldn't return anything. Also the docblock for this method is clearly from some other method. :)

Reply:
Sloppy, sloppy... https://github.com/Automattic/newspack-plugin/pull/126/commits/ad64293d40396d7e251d9a4411e1e4137fe3f398